### DISCUSSION TITLE
Deprecating and redirecting from one tap to another

### DISCUSSION THREAD

**Discussion by anthony-c-martin** [Homebrew/discussions#1253] [Tap maintenance and Homebrew development] [answered]
We've got two active taps, and would like to deprecate one of them. Preferably I'd like to either automatically redirect existing users of the old tap to the new one, or to give them an error message with info on how to migrate.

Is there any way to handle this within homebrew?

Here's our old formula: https://github.com/Azure/bicep/blob/main/Formula/bicep.rb
Here's our new formula: https://github.com/Azure/homebrew-bicep/blob/main/Formula/bicep.rb

**Comment by carlocab** [comment, marked as answer]
Try this: https://docs.brew.sh/Migrating-A-Formula-To-A-Tap

**Reply by anthony-c-martin** [reply]
Thanks for the pointer!
